### PR TITLE
fix BELGIUM_MAXIMUM_DEMAND_MONTH regex

### PR DESCRIPTION
**Pull request by koen-dejonghe**
Regex for BELGIUM_MAXIMUM_DEMAND_MONTH will also match strings intended for BELGIUM_MAXIMUM_DEMAND_13_MONTHS.
This results in an error, for example:
```
ignore line with signature \d-\d:1\.6\.0.+?\r\n, because parsing failed.
Traceback (most recent call last):
  File "/opt/homebrew/anaconda3/envs/dsmr-mqtt/lib/python3.10/site-packages/dsmr_parser/parsers.py", line 90, in parse
    dsmr_object = parser.parse(match)
  File "/opt/homebrew/anaconda3/envs/dsmr-mqtt/lib/python3.10/site-packages/dsmr_parser/parsers.py", line 226, in parse
    values=self._parse(line)
  File "/opt/homebrew/anaconda3/envs/dsmr-mqtt/lib/python3.10/site-packages/dsmr_parser/parsers.py", line 198, in _parse
    raise ParseError("Invalid '%s' line for '%s'", line, self)
dsmr_parser.exceptions.ParseError: ("Invalid '%s' line for '%s'", '1-0:1.6.0)(1-0:1.6.0)(230101000000W)(221209193000W)(10.000*kW)(230201000000W)(230130184500W)(05.000*kW)\r\n', <dsmr_parser.parsers.MBusParser object at 0x1021aa170>)
```

We can avoid this by adding a negative lookbehind on '(' to the regex of BELGIUM_MAXIMUM_DEMAND_MONTH.

### CODE REVIEW

File: dsmr_parser/obis_references.py [hunk 1]
```diff
@@ -82,7 +82,7 @@
 BELGIUM_VERSION_INFORMATION = r'\d-\d:96\.1\.4.+?\r\n'
 BELGIUM_EQUIPMENT_IDENTIFIER = r'\d-0:96\.1\.1.+?\r\n'
 BELGIUM_CURRENT_AVERAGE_DEMAND = r'\d-\d:1\.4\.0.+?\r\n'
-BELGIUM_MAXIMUM_DEMAND_MONTH = r'\d-\d:1\.6\.0.+?\r\n'
+BELGIUM_MAXIMUM_DEMAND_MONTH = r'(?<!\()\d-\d:1\.6\.0.+?\r\n' # negative lookbehind on ( to avoid conflict with BELGIUM_MAXIMUM_DEMAND_13_MONTHS
```
Comment (dupondje) [inline dsmr_parser/obis_references.py:85]: Would using 
```
^\d-\d:1\.6\.0.+?\r\n
```
not be enough?

Comment (dupondje) [reply dsmr_parser/obis_references.py:85]: @koen-dejonghe ? Could you check please?

Comment (koen-dejonghe) [reply dsmr_parser/obis_references.py:85]: No, `^\d-\d:1\.6\.0.+?\r\n` does not match anything.

**Comment by ndokter** [comment]
Is it somehow reproducable in this unit test? `TelegramParserFluviusTest`

**Comment by dupondje** [comment]
Both issues fixed in #132

**Pull request closed without merging**